### PR TITLE
PML/SPML/UCX: added evaluation of mmap events

### PR DESCRIPTION
**Pull request by hoopoepg**
- there was a set of UCX related issues reported which caused
  by mmap API hooks conflicts. We added diagnostic of such
  problems to simplify bug-resolving pipeline

Signed-off-by: Sergey Oblomov <sergeyo@mellanox.com>

### CODE REVIEW

**Comment by hoopoepg** [comment]
bot:mellanox:retest

**Comment by hoopoepg** [comment]
@yosefe this fix + https://github.com/openucx/ucx/pull/3368 detects issue
what do you think?

File: opal/mca/common/ucx/common_ucx.c [hunk 1]
```diff
@@ -132,6 +132,28 @@ static void opal_common_ucx_mca_fence_complete_cb(int status, void *fenced)
     *(int*)fenced = 1;
 }
 
+void opal_common_ucx_mca_proc_added(void)
+{
+#if HAVE_DECL_UCM_TEST_EVENTS
+    static int warned = 0;
+    static char *mem_hooks_suggestion = "Try to add command line agrument "
```
Comment (yosefe) [inline opal/mca/common/ucx/common_ucx.c]: ` Pls try adding  --mca opal_common_ucx_opal_mem_hooks 1 to mpirun/oshrun command line to resolve this issue`

File: opal/mca/common/ucx/common_ucx.h [hunk 2]
```diff
@@ -49,6 +49,11 @@ BEGIN_C_DECLS
                         __FILE__ ":" MCA_COMMON_UCX_QUOTE(__LINE__) \
                         " Error: " __VA_ARGS__)
 
+#define MCA_COMMON_UCX_WARN(...)                                    \
+    opal_output_verbose(0, opal_common_ucx.output,                  \
```
Comment (yosefe) [inline opal/mca/common/ucx/common_ucx.h]: can it call MCA_COMMON_UCX_VERBOSE(0, ..) ?

File: opal/mca/common/ucx/common_ucx.c (hunk 1 above)
Comment (hoopoepg) [reply opal/mca/common/ucx/common_ucx.c]: message is updated

File: opal/mca/common/ucx/common_ucx.h (hunk 2 above)
Comment (hoopoepg) [reply opal/mca/common/ucx/common_ucx.h]: macro is fixed

**Pull request merged**